### PR TITLE
Add gradle

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: docshtml/KIDLspec.css
Comment (MrCreosote) [inline docshtml/KIDLspec.css:1]: This is generated by kb-sdk compile

File: docshtml/NarrativeMethodStore.html
Comment (MrCreosote) [inline docshtml/NarrativeMethodStore.html:1]: Generated etc.

File: lib/biokbase/narrative_method_store/authclient.py
Comment (MrCreosote) [inline lib/biokbase/narrative_method_store/authclient.py:1]: Not necessary for clients

**Review by Xiangs18 — APPROVED** [review]
LGTM!

**Pull request merged**